.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -31,23 +31,32 @@ Once all issues have passed acceptance testing and have been merged into `master
+  - [ ] Add the release to the [Supported versions](https://docs.communityhealthtoolkit.org/core/releases/) and update the EOL date and status of previous releases. Also add a link in the `Release Notes` section to the new release page.
+  - [ ] Assign the PR to:
+    - The Director of Technology
+    - An SRE to review and confirm the documentation on upgrade instructions and breaking changes is sufficient
 - [ ] Until release testing passes, make sure regressions are fixed in `master`, cherry-pick them into the release branch, and release another beta.
 - [ ] Create a release in GitHub from the release branch so it shows up under the [Releases tab](https://github.com/medic/cht-core/releases) with the naming convention `<major>.<minor>.<patch>`. This will create the git tag automatically. Link to the release notes in the description of the release.
 - [ ] Confirm the release build completes successfully and the new release is available on the [market](https://staging.dev.medicmobile.org/builds/releases). Make sure that the document has new entry with `id: medic:medic:<major>.<minor>.<patch>`
 - [ ] Review the scalability results on S3 at medic-e2e/scalability/$TAG_NAME. Add the release `.jtl` file to `cht-core/tests/scalability/previous_results`. Compare the trend using the [scalability documentation](https://github.com/medic/cht-core/blob/master/tests/scalability/README.md).
 - [ ] Upgrade the `demo-cht.dev` instance to this version.
-- [ ] Add the release to the [Supported versions](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) and update the EOL date and status of previous releases.
 - [ ] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/c/product/releases/26), under the "Product - Releases" category using this template:
 ```
-@channel *We're excited to announce the release of {{version}} of {{product}}*
+*We're excited to announce the release of {{version}} of {{product}}*
 
 New features include {{key_features}}. We've also implemented loads of other improvements and fixed a heap of bugs.
 
-Read the release notes for full details: {{url}}
+Read the [release notes]({{url}}) for full details.
 
-Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our software support documentation: https://docs.communityhealthtoolkit.org/core/overview/supported-software/
+Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our [software support documentation](https://docs.communityhealthtoolkit.org/core/overview/supported-software/).
 
-See what's scheduled for the next releases: https://github.com/medic/cht-core/milestones
+Check our [milesones](https://github.com/medic/cht-core/milestones) to see what's scheduled for the next releases.
```

Comment:
I think this should probably point to the roadmap instead, or as well, as the milestones...

```suggestion
Check out our [roadmap](https://github.com/orgs/medic/projects/112) to see what we're working on next.
```